src/sentry/auth/superuser.py

```diff
@@ -73,6 +78,15 @@ class Superuser:
 
     org_id = ORG_ID
 
+    def _check_expired_on_org_change(self):
+        if self.expires is not None:
+            session_start_time = self.expires - MAX_AGE
+            current_datetime = timezone.now()
+            if current_datetime - session_start_time > MAX_AGE_PRIVILEGED_ORG_ACCESS:
+                self.set_logged_out()
```

Comment:
should we add a `logger.warning` here with a similar message to the ones in `get_session_data`?

Reply:
yup that sgtm